### ISSUE TITLE
`R CMD build` timeouts on some Bioconductor packages with recent devel version of BiocStyle

### ISSUE THREAD

**Issue by hpages** [Bioconductor/BiocStyle#31]
Hi Andrzej,

We recently started to observe `R CMD build` timeouts on some Bioconductor packages (e.g. TRONCO):

  https://bioconductor.org/checkResults/3.6/bioc-LATEST/

They seem to be caused by some recent changes to `BiocStyle::latex()`. If I install the release version of BiocStyle then `R CMD build TRONCO` finishes in less than 1 minute.

Thanks,
H.

**Comment by aoles** [comment]
Hi Hervé,
thank you for drawing my attention to this, so far I was mostly chasing after packages failing with an ERROR. I will look into potentially affected packages, which apart from *TRONCO* seem to include *CINdex*, *DMRcate*, *GenomicRanges*, *groHMM* and *sampleClassifier*.

**Comment by lshep** [comment]
Yes there seem to be several packages that now have latex ERRORs that seem to be related to the change with BiocStyle::latex().   Some of the packages possibly affected failing with latex ERRORS include _blima, DaMiRseq, DiffBind, funtooNorm, Linnorm, MBttest, MutationalPatterns,NanoStringQCPro,OncoScore, OperaMate, qcmetrics, quantro, rCGH_, and _RiboProfiling_.   I haven't tried all of the package but at least OperaMate and DaMiRseq fail using the devel version of BiocStyle but build correctly using the release version of BiocStyle.

**Comment by aoles** [comment]
Hi Lori,
thank you! I'm actually well aware of these problems. In fact, I've been monitoring the build reports since the switch in BiocStyle and I have already fixed around 25 failing packages, most through direct pushes. As for the remaining issues: last week I contacted the authors of *blima* and *qcmetrics* providing a solution to their specific build issues. The rest of the packages you listed usually fails because their vignettes use non-standard ways of specifying author affiliations, or contain other LaTeX customizations which we don't support. The updated `BiocStyle::latex()` leverages the *authblk* LateX package to accommodate complex scenarios, but unfortunately it is also more fragile than the plain LaTeX macros. 

Once I identify a complete list of the affected packages I will post an announcement on the devel mailing list asking their maintainers for an action.

@hpages in case you're wondering, I've already fixed *GenomicRanges*.
